fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -1559,56 +1559,31 @@ private String getVersionIdFromETagValue(String ifMatchValue) {
     private Bundle processBundleEntries(Bundle requestBundle, Map<Integer, Entry> validationResponseEntries, boolean skippableUpdates) throws Exception {
         log.entering(this.getClass().getName(), "processBundleEntries");
 
-        FHIRTransactionHelper txn = null;
-
         // Generate a request correlation id for this request bundle.
         bundleRequestCorrelationId = UUID.randomUUID().toString();
         if (log.isLoggable(Level.FINE)) {
             log.fine("Processing request bundle, request-correlation-id=" + bundleRequestCorrelationId);
         }
 
         try {
-            BundleType.Value bundleType = requestBundle.getType().getValueAsEnum();
-            // If we're working on a 'transaction' type interaction, then start a new transaction now
-            if (bundleType == BundleType.Value.TRANSACTION) {
-                txn = new FHIRTransactionHelper(getTransaction());
-                txn.begin();
-                if (log.isLoggable(Level.FINE)) {
-                    log.fine("Started new transaction for transaction bundle, request-correlation-id="
-                        + bundleRequestCorrelationId);
-                }
-            }
-
             // Build a mapping of local identifiers to external identifiers for local reference resolution.
             Map<String, String> localRefMap = buildLocalRefMap(requestBundle, validationResponseEntries);
 
             // Process entries.
+            BundleType.Value bundleType = requestBundle.getType().getValueAsEnum();
             List<Entry> responseEntries = processEntriesByMethod(requestBundle, validationResponseEntries,
-                    txn != null, localRefMap, bundleRequestCorrelationId, skippableUpdates);
+                    bundleType == BundleType.Value.TRANSACTION, localRefMap, bundleRequestCorrelationId, skippableUpdates);
 
             // Build the response bundle.
+            // TODO add support for posting history bundles
             Bundle.Builder bundleResponseBuilder = Bundle.builder().entry(responseEntries);
             if (bundleType == BundleType.Value.BATCH) {
                 bundleResponseBuilder.type(BundleType.BATCH_RESPONSE);
             } else if (bundleType == BundleType.Value.TRANSACTION) {
                 bundleResponseBuilder.type(BundleType.TRANSACTION_RESPONSE);
-            } else {
-                // TODO add support for posting history bundles
-                String msg = "Bundle.type must be either 'batch' or 'transaction'.";
-                throw buildRestException(msg, IssueType.VALUE);
```

Comment:
did you confirm that we do this validation somewhere else now?  i can't remember if we do it in the validate(Bundle) method or not...

Reply:
yes, we do it in validateBundle() on line 1260